src/service.ts

```diff
@@ -143,6 +146,14 @@ export default class TVLabsService implements Services.ServiceInstance {
     }
   }
 
+  private injectAuthorizationHeader() {
+    this._config.headers = this._config.headers || {};
+
+    if (!this._config.headers.Authorization) {
+      this._config.headers.Authorization = `Bearer ${this._options.apiKey}`;
```

Comment:
```suggestion
      this._config.headers.Authorization = `Bearer ${this.apiKey()}`;
```

We can re-use this getter